src/main/java/org/cascadebot/cascadebot/commands/music/PlayCommand.java

```diff
@@ -17,6 +18,16 @@
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
+        VoiceChannel voiceChannel = sender.getVoiceState().getChannel();
+
+        if (voiceChannel != null) {
+            if (context.getMusicPlayer().getConnectedChannel() != voiceChannel) {
+                context.runOtherCommand("join", sender, context);
+            }
+        } else {
+            context.runOtherCommand("join", sender, context);
+        }
```

Comment:
We need to add a setting to enable this behaviour. Maybe a `joinonplay` in MusicSettings

Reply:
Just committed. Did I do it right?